### PR TITLE
do not show if values are empty

### PR DESCRIPTION
**Pull request by hmert**

### CODE REVIEW

File: src/Ojs/CoreBundle/Service/Twig/OjsExtension.php [hunk 1]
```diff
@@ -539,10 +539,18 @@ public function getAuthorsInfo(Author $author)
 
         $text = '
         <p id="author$' . $author->getId() . '">
-        <b>' . $this->translator->trans('author') . ': </b>' . $fullName . '</br>
-        <b>' . $this->translator->trans('email') . ': </b>' . $email . '</br>
-        <b>' . $this->translator->trans('institution') . ': </b>' . $institution . '</br>
-        <b>' . $this->translator->trans('country') . ': </b>' . $author->getCountry() . '</p><hr>';
+        <b>' . $this->translator->trans('author') . ': </b>' . $fullName . '</br>';
+
+        if (!empty($this->translator->trans('email'))){
```
Comment (behram) [inline src/Ojs/CoreBundle/Service/Twig/OjsExtension.php]: probably you want to check if empty `$email` variable. why did you checked translation existance? @hmert

File: src/Ojs/CoreBundle/Service/Twig/OjsExtension.php [hunk 2]
```diff
@@ -539,10 +539,18 @@ public function getAuthorsInfo(Author $author)
...
+            $text .= '<b>' . $this->translator->trans('email') . ': </b>' . $email . '</br>';
+        }
+        if (!empty($this->translator->trans('institution'))){
```
Comment (behram) [inline src/Ojs/CoreBundle/Service/Twig/OjsExtension.php]: same as upper

File: src/Ojs/CoreBundle/Service/Twig/OjsExtension.php (hunk 1 above)
Comment (hmert) [inline src/Ojs/CoreBundle/Service/Twig/OjsExtension.php]: Oh right!

**Comment by behram** [comment]
looks like there is no problem i am merging now.

**Pull request merged**